openpype/hosts/maya/plugins/create/create_rig.py

```diff
@@ -20,6 +20,13 @@ def create(self, subset_name, instance_data, pre_create_data):
         instance_node = instance.get("instance_node")
 
         self.log.info("Creating Rig instance set up ...")
+        # TODO：change name (_controls_set -> _rigs_SET)
```

Comment:
Honestly I wouldn't find `_rigs_SET` clearer than `_controls_SET` since it should only include rig controllers so I personally wouldn't recommend this rename.
```suggestion
        # TODO：change name (_controls_SET -> _rigs_SET)
```